wgpu-hal/src/dx12/mod.rs

```diff
@@ -541,11 +545,23 @@ pub struct Surface {
     target: SurfaceTarget,
     supports_allow_tearing: bool,
     swap_chain: RwLock<Option<SwapChain>>,
+    options: wgt::Dx12BackendOptions,
 }
 
 unsafe impl Send for Surface {}
 unsafe impl Sync for Surface {}
 
+impl Surface {
+    /// Returns the waitable handle of the swapchain, if it exists.
```

Comment:
We should also define the lifetime of the handle

Reply:
Define the lifetime in a SAFETEY comment or as actual Rust lifetimes?

Reply:
Just as a safety comment, saying it's valid until the surface is reconfigured or unconfigured or whatever